my_code.py

```diff
@@ -0,0 +1,33 @@
+import numpy as np
+import matplotlib
+import os
+from astropy.io import fits
+import glob
+data = fits.getdata('/Users/bostroem/Desktop/images/hubble_img.fits')
+col1 = data[20, :] 
+col2 = data[200, :]
+col3 = data[800, :]
```

Comment:
The desired values could be saved into a list:
```python
desidred_columns = [20, 200, 800]
for column in desired_columns:
    myrow = data[column, :]
```
Or provide meaningful names instead of `colx`